src/containers/access/src/EditAccessRequestContainer.js

```diff
@@ -27,29 +35,53 @@ const PrintButton = styled(IconButton)`
 `;
 
 const EditAccessRequestContainer = () => {
-  const data = useSelector(selectAccessRequestData());
+  const dispatch = useDispatch();
   const componentRef = useRef();
+  const data = useSelector(selectAccessRequestData());
   const handlePrint = useReactToPrint({
     content: () => componentRef.current,
   });
-  const requestState = useSelector((s) => s.getIn([ACCESS, UPDATE_ACCESS_REQUEST, REQUEST_STATE]));
 
-  const accessId = getEntityKeyId(data) || '';
-  const isSubmitting = isPending(requestState);
+  const updateState = useSelector((s) => s.getIn([ACCESS, UPDATE_ACCESS_REQUEST, REQUEST_STATE]));
+  const fetchState = useSelector((s) => s.getIn([ACCESS, GET_ACCESS_REQUEST, REQUEST_STATE]));
+
+  const match :Match = useRouteMatch();
+  const accessId = match?.params?.accessId || '';
```

Comment:
Why not use `getParamFromMatch`?

```
import { RoutingUtils } from 'lattice-utils';
const { getParamFromMatch } = RoutingUtils;
const requestId = getParamFromMatch(match, Routes.REQUEST_ID_PARAM) || '';
```

Reply:
1) didn't know it existed
2) this is actually less code

Reply:
Less code isn't necessarily a good metric, and I'm not convinced that's actually true. `match?.params?.accessId` will expand to something a bit larger when it's transpiled. Also, I would argue `getParamFromMatch` is a bit safer because you're passing a constant into it. If `accessId` ever changes, you'll have to make sure to change all uses of it whereas that's not the case if you use `getParamFromMatch`.

Reply:
```
const accessId = (match === null || match === void 0 ? void 0 : (_match$params = match.params) === null || _match$params === void 0 ? void 0 : _match$params.accessId) || '';
```